spec.bs

```diff
@@ -185,6 +185,7 @@ dictionary GenerateBidInterestGroup {
   USVString biddingWasmHelperURL;
   USVString updateURL;
   USVString trustedBiddingSignalsURL;
+  DOMString trustedBiddingSignalsSlotSizeMode = "none";

```

Comment:
I'm not comfortable doing that - the current description looks inaccurate (e.g., execution mode is stored as an enum, so currently contributes 4 bytes, rather than the string length).  This follows the same pattern.  I'm not comfortable documenting things directly contrary to what the code does with no plans to make them match behavior, and I think it's unclear what is the correct behavior.

Reply:
I've gone ahead and added an inaccurate description of the IG's size logic for trustedBiddingSignalsSlotSizeMode, matching the executionMode stuff.